ui/src/config/section/offering.js

```diff
@@ -215,6 +215,15 @@ export default {
         listView: true,
         popup: true,
         component: () => import('@/views/offering/ImportBackupOffering.vue')
+      }, {
+        api: 'updateBackupOffering',
+        icon: 'edit',
+        label: 'label.edit',
+        dataView: true,
+        groupAction: true,
```

Comment:
You don't want groupable action on edit/updating offering, otherwise lgtm

Reply:
@SadiJr same thinking, don't set `groupAction` for edit/update

Reply:
@rhtyd @utchoang Done, thanks for the suggestion.